bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -1,590 +1,72 @@
-# add-on
```

Comment:
@markus7017 is doing an extensive refactoring of this binding, maybe he can confirm that this is ok? Removing ~600 lines and only adding ~75 lines of translation.

Reply:
We are just finalizing the latest PR. Anything open from this PR?

Reply:
@florian-h05 I always manage translations incl. using crowdin
There are still some strings in the XML like symbolic thing name, which I didn't move to the resource file. I think it doesn't make sense to have the same string in EN+DE, but of course I could do that.

I didn't got what's required on top, is there anything specific from this PR

Reply:
As you always manage translations, I will leave those translations to you. 
I guess I will close this PR now as it seems no one is willing to review it, so do your Shelly PR as you want to.

> I think it doesn't make sense to have the same string in EN+DE, but of course I could do that.

Agreed. I haven’t checked these details when updating the translations, because doing that for all bindings would be very time-expensive job.